### ISSUE TITLE
Would you mind adding a brief README?

### ISSUE THREAD

**Issue by Tr4cer0** [vanhoefm/dragonforce#1]
Hi, would you mind adding a brief README for reproduce the experiment?

Thanks a lot.

**Comment by neildalal** [comment]
Yes please, a brief readme would be really helpful to recreate the attack. 
Could the author please add one? Thank you !
